tests/s3_api_gradle.erl

```diff
@@ -0,0 +1,68 @@
+-module(s3_api_gradle).
+
+-behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+    confirm/0
+]).
+
+-define(NODE_COUNT, 3).
+-define(TEST_USER, ["test", "test@example.com"]).
+-define(METADATA_KEY, [{<<"security">>, <<"users">>}, <<"test">>]).
+
+confirm() ->
+    %% Build a simple cluster
+    Nodes = rt:build_cluster(?NODE_COUNT),
```

Comment:
We'll need to force the use of the `riak_kv_elevedb_backend` in this test before long, because only that backend will (initially at least) support the necessary `grouped_list_keys` functionality we'll need to do a good list keys test.

This can be accomplished by passing a Config parameter to build_cluster that includes the backend specifically - something like this:

```erlang
 Config = [{riak_kv, [
        {storage_backend, Backend}]}
    ],
  Nodes = rt:build_cluster(?NODE_COUNT, Config),
...
```